docs/modules/ROOT/pages/maintenance_process.adoc

```diff
@@ -0,0 +1,78 @@
+= Maintenance Process
+
+== Overview
+This page provides information about our maintenance process.
+Basically we have two kind of maintenances procedures.
+For the APPUiO clusters and Openshift clusters our internal Squad _Sirius_ is responsible.
```

Comment:
I guess people reading the KB don't want to know about Squads. I propose to leave this paragraph out, as we provide maintenance "as VSHN" and it shouldn't make any difference which Squad does the work.